Tests/test_PDB.py

```diff
@@ -1332,6 +1332,23 @@ def test_NACCESS_asa_file(self):
             naccess = process_asa_data(asa)
         self.assertEqual(len(naccess), 524)
 
+class TestPDBIO(unittest.TestCase):
+    
+    def test_example(self):
+        p = PDBParser(PERMISSIVE=True)
+        filenumber, filename = tempfile.mkstemp()
+        os.close(filenumber)
+        io = PDBIO()
+        io.set_structure(s)
```

Comment:
You have not defined ``s`` yet, the original code did:

```
s = p.get_structure("test", sys.argv[1])
```

However, you need to pick an example PDB file from under ``Tests/PDB/*.pdb``